created missing telnyx explanation

yumminova opened the pull request.

sergey-safarov commented on core/kazoo_numbers/doc/knm_telnyx:
```diff
@@ -0,0 +1,55 @@
+
+To enable kazoo number manager to use the telnyx module you will need to first update the `number_manager` document in the `system_config` database.
+
+In the configuration section that applies to your crossbar servers (or the default) add `"knm_voxbone"` to the `carrier_modules` parameter.
```
Why Voxbone reference present in knm_telnyx documentation?

yumminova replied:
Good question, let me remove that

yumminova replied:
i investigated the situation, and i have come to the conclusion that it must have been a typo.

sergey-safarov commented on core/kazoo_numbers/doc/knm_telnyx:
```diff
@@ -0,0 +1,55 @@
+   "default": {
+       "debug": "true",       
+       "enable_provisioning": "true",
+       "monthly_recurring_cost": "1",
```
no exist on telnyx module

yumminova commented:
i received en email with commment: no exist on telnyx module

But i dont know what u mean, there are no docs or examples, and this works.
So if u know how that doc should look, feel free to change it or show me the way

yumminova commented:
https://github.com/2600hz/kazoo-crossbar/blob/master/priv/couchdb/schemas/system_config.number_manager.telnyx.json

Thats where i got the fields, let me know if that is not correct

sergey-safarov commented:
yes, I wrote false comment and then deleted.
For me PR looks fine

yumminova commented:
Ok, that why I didn't see it)) I'm not a github user so was confused.

Thanks

jamesaimonetti commented:
@yumminova big thanks again! I cleaned up the file a bit and added it to the documentation index. Should be live at https://docs.2600hz.com/dev/core/kazoo_numbers/doc/knm_telnyx/

yumminova commented:
I'm a captain, so thanks for cleaning up a little))
I barely know how to do it, but improving 😊

Verzonden met ProtonMail Mobile

-------- Oorspronkelijk bericht --------
Aan 8 jan. 2020 01:44, James Aimonetti wrote:

> [@yumminova](https://github.com/yumminova) big thanks again! I cleaned up the file a bit and added it to the documentation index. Should be live at https://docs.2600hz.com/dev/core/kazoo_numbers/doc/knm_telnyx/
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, [view it on GitHub](https://github.com/2600hz/kazoo/pull/6234?email_source=notifications&email_token=ACA6RATEWZ2MEH7Y5IQKA4LQ4UOYTA5CNFSM4KCWVBS2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIKZIFA#issuecomment-571839508), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/ACA6RART3KB4KUFWTI37N73Q4UOYTANCNFSM4KCWVBSQ).
